docs/index.rst

```diff
@@ -12,6 +12,14 @@ Welcome to Stellio's documentation!
 
    source/index
 
+Migrations notes
+===================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   source/migrations/index
```

Comment:
A migration action must be associated to a specific version (you always upgrade from a version). Either people will never know if they have to do it or not.